src/ImageSharp.Web/FormatUtilities.cs

```diff
@@ -19,8 +19,7 @@ namespace SixLabors.ImageSharp.Web
     /// </summary>
     public sealed class FormatUtilities
     {
-        private readonly IImageFormat[] imageFormats;
-        private readonly Dictionary<IImageFormat, string[]> fileExtensions = new Dictionary<IImageFormat, string[]>();
+        private readonly List<string> fileExtensions = new List<string>();
         private readonly Dictionary<string, string> fileExtension = new Dictionary<string, string>();
```

Comment:
Should it be renamed `extensionsByMimeType` to make the code clearer? It's hard to understand what it is otherwise, and the two collections are very close.

Reply:
Agreed. My original naming is bad. We can do a bit of cleanup here actually. No need to store the `imageFormats` array now and should use C# 9 initializer for the fields to make things more terse.

Reply:
Updated.